docker/docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+# note: we do not run prod service with docker-compose, we use it just for
+# configuring the production build
+services:
+  # base service, exists to hold common config, but is not actually used directly
+  base:
+    build:
+      context: ..
+      # path relative to context
+      dockerfile: docker/Dockerfile
+      # the production stage in the Dockerfile
+      target: new-project-prod # CHANGE ME!
+      args:
+        # this makes the image work for later cache_from: usage
+        - BUILDKIT_INLINE_CACHE=1
+        # env vars supplied by just
+        - BUILD_DATE
+        - GITREF
+    # use dockers builitin PID daemon
+    init: true
+    ports:
+      - "8000:8000"
+
+  # prod service. Used to build prod image and run/test it locally
+  prod:
+    # image name, both locally and public
+    image: new-project # CHANGE ME!
+    extends:
+      service: base
+    volumes:
+      - ../workspaces:/workspaces
```

Comment:
This isn't a folder that automatically exists in every project. Maybe a comment here about replacing it with volumes specific to the project? And/or a note in the README?